release.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python3
 import argparse
+from github import Github
```

Comment:
This `github` package likely isn't installed on everyone's dev machines - it'd be slightly less of a hassle to curl the releases API using the built-in requests lib